README.md

```diff
@@ -1,135 +1,238 @@
-Chitter Challenge
-=================
+# Chitter Challenge
+A small Twitter clone that will allow users to post messages (peeps) to a public stream
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+## Motivation
+This project is an unguided, solo, understanding check of TDD, OOD, MVC, of a web application with a database.
 
-Challenge:
--------
+## Build status
+[![Build Status](https://travis-ci.com/chriswhitehouse/chitter-challenge.svg?branch=master)](https://travis-ci.com/chriswhitehouse/chitter-challenge)
 
-As usual please start by forking this repo.
+## Code style
+[![Ruby Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://github.com/testdouble/standard)
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+## Screenshots
+Include logo/demo screenshot etc.
 
-Features:
--------
+## Tech/framework used
+Ruby with Sinatra web framework, and PostgreSQL database. Tested using Rspec, and Capybara DSLs with Rubocop and Simplecov.
 
+## Features
+### User Story 1
 ```
-STRAIGHT UP
-
 As a Maker
 So that I can let people know what I am doing  
 I want to post a message (peep) to chitter
+```
 
+![Class Diagram](https://github.com/chriswhitehouse/chitter-challenge/blob/master/diagrams/user_story_1/class.svg)
+![Entity](https://github.com/chriswhitehouse/chitter-challenge/blob/master/diagrams/user_story_1/entity.svg)
+![Sequence](https://github.com/chriswhitehouse/chitter-challenge/blob/master/diagrams/user_story_1/sequence.svg)
+
```

Comment:
these diagrams are incredible! Very clear and useful in understanding your thought processes